students/taejun/users/views.py

```diff
@@ -55,3 +55,21 @@ def post(self, request):
             return JsonResponse({'message': 'VALIDATION_ERROR'}, status=400)
 
         return JsonResponse({'message': 'CREATED'}, status=201)
+
+
+class SigninView(View):
+
+    def post(self, request):
+
+        data = json.loads(request.body)
+
+        try:
+            email    = data['email']
+            password = data['password']
+            User.objects.get(email=email, password=password)
+            return JsonResponse({'message': 'SUCCESS'}, status=200)
+
+        except User.DoesNotExist:
+            return JsonResponse({'message': 'INVALID_USER'}, status=401)
+        except KeyError:
+            return JsonResponse({'message': 'KEY_ERROR'}, status=400)
```

Comment:
1. 빈 줄 지워주세요!
2. Alignment 맞춰주세요!
3. `User.objects.get(email=email, password=password)` 보다는 `exists()` method를 사용해서 객체의 존재 여부를 코드 가독성을 높이는게 어떨까요?

```suggestion
class SigninView(View):
    def post(self, request):
        data = json.loads(request.body)

        try:
            email    = data['email']
            password = data['password']
            
            if not User.objects.filter(email=email, password=password).exists():
                return JsonResponse({'message': 'INVALID_USER'}, status=404)
                
            return JsonResponse({'message': 'SUCCESS'}, status=200)
                        
        except KeyError:
            return JsonResponse({'message': 'KEY_ERROR'}, status=400)
```

Reply:
LGTM!!